gradle/libs.versions.toml

```
@@ -19,7 +19,7 @@ netty = "4.2.4.Final"
 netty-tcnative = "2.0.72.Final"
 
 jetty = "9.4.58.v20250814"
-jetty-jakarta = "11.0.26"
+jetty-jakarta = "12.0.21"
```

Let's update the version
```suggestion
jetty-jakarta = "12.0.25"
```

---

Oh yes, I had already done this but forgot to push the change 🤦